cmd/keepalived-exporter/main.go

```diff
@@ -50,6 +51,9 @@ func main() {
 	keepalivedCollector := collector.NewKeepalivedCollector(*keepalivedJSON, *keepalivedCheckScript, c)
 	prometheus.MustRegister(keepalivedCollector)
 
+	promver.Version = version
```

Comment:
Thanks! Just two things are remaining to get it done.
1. Please remove the vars (`version`, `commit`, `buildTime`) from the main package and edit the Makefile to set directly to the `prometheus/common/version` package vars.
2. Please edit the Makefile to set `branch` and `revision` as well (Let's not expose a metric with empty labels).